Tests/SnapshotTestingTests/SnapshotTestingTests.swift

```diff
@@ -825,6 +825,14 @@ final class SnapshotTestingTests: XCTestCase {
     #endif
   }
   
+  func testUIViewFailsIfNewViewHasNoFrame() {
```

Comment:
who tests the testing test tests?

Reply:
![giphy-2](https://user-images.githubusercontent.com/2341305/150874672-f9b5bffc-a072-41e3-a6cd-2cf3e26d9068.gif)

Reply:
![giphy](https://user-images.githubusercontent.com/91489953/150875479-6d6cc567-737d-476e-94b6-419aa8b13f65.gif)